paimon-common/src/main/java/org/apache/paimon/lookup/sort/Footer.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.paimon.lookup.sort;
+
+import org.apache.paimon.memory.MemorySlice;
+import org.apache.paimon.memory.MemorySliceInput;
+import org.apache.paimon.memory.MemorySliceOutput;
+
+import javax.annotation.Nullable;
+
+import java.io.IOException;
+
+import static org.apache.paimon.lookup.sort.SortLookupStoreWriter.MAGIC_NUMBER;
+import static org.apache.paimon.utils.Preconditions.checkArgument;
+
+/** Footer for a sorted file. */
+public class Footer {
+
+    public static final int ENCODED_LENGTH = 36;
+
+    @Nullable private final BloomFilterHandle bloomFilterHandle;
+    private final BlockHandle indexBlockHandle;
```

Comment:
A version field should be reserved here so that we can easily upgrade the version number in the future and be compatible with old versions?

Reply:
Is magic number something like version? Maybe we can add version field too.

Reply:
It seems a little different. Currently, `magic_number` is unique and unchanged, mainly used to verify whether the data file is correct; `version` may have multiple versions as we upgrade the storage structure, and different data processing will be performed according to different `version`s.

Reply:
Sounds good to me.